src/Conversion/ONNXToKrnl/Math/Elementwise.cpp

```diff
@@ -91,6 +91,36 @@ struct ScalarOp<ONNXSqrtOp> {
   using IOp = SqrtOp; // not use
 };
 
+template <>
+struct ScalarOp<ONNXAtanOp> {
+  using FOp = AtanOp;
+  using IOp = AtanOp; // not use
```

Comment:
What does "not use" mean?

Reply:
It should be "not used"; the struct request one op for int, one for float. Some operations exist only for int, others only for float, some for both. Comment indicate which one is not used, and should never be called

Reply:
Thanks for the clarification! Makes sense.